src/cron/arweave/mod.rs

```diff
@@ -110,8 +109,6 @@ pub enum ArweaveProtocol {
 #[derive(Clone)]
 pub struct Arweave {
     pub host: String,
```

Comment:
Should this be `Uri` so that later in the code there is no need to parse and validate this. Then could implement `Deref` for `Target = Uri` so that you can simply deref the value when needed.

Reply:
Agreed.